src/components/RegisterForm/RegisterForm.jsx

```diff
@@ -3,13 +3,23 @@ import * as yup from 'yup';
 import { register } from '@/redux-store/AuthOperations/AuthOperations';
 import { useDispatch } from 'react-redux';
 import {
+  Background,
+  RegisterFormContainer,
+  SignInFormStyles,
+  SignInTitle,
+  SignInText,
+  LoginLink,
   ItemWrapp,
   StyledLabel,
   Error,
   Success,
   // ValidationIcon,
   StyledInput,
   SignUpBtn,
+  Separator,
+  ButtonBlock,
+  ButtonGoogle,
+  ButtonFacebook,
 } from './RegisterForm.styled';
 
 const schema = yup.object().shape({
```

Comment:
This schema object must me moved outside of the component or in a defferent file. We don't need to excute it on every component render.